pkg/volume/csi/csi_attacher.go

```diff
@@ -352,9 +352,8 @@ func (c *csiAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMo
 
 	if err != nil {
 		glog.Errorf(log("attacher.MountDevice failed: %v", err))
-		if err := removeMountDir(c.plugin, deviceMountPath); err != nil {
-			glog.Error(log("attacher.MountDevice failed to remove mount dir after a NodeStageVolume() error [%s]: %v", deviceMountPath, err))
-			return err
+		if clearErr := removeMountDir(c.plugin, deviceMountPath); clearErr != nil {
```

Comment:
nit: `clearErr` should be a more relevant name. `removeMountDirErr`?

Reply:
Updated `clearErr` to `removeMountDirErr`, PTAL.